config/rbac/role.yaml

```diff
@@ -89,6 +89,12 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - security.openshift.io
```

Comment:
This is OpenShift specific so we need to keep it separate and outside the normal installation PATH. I think issue #46 would help resolve this more easily.